api/go/routers.go

```diff
@@ -54,15 +54,33 @@ func NewRouter(routers ...Router) *mux.Router {
 				Path(route.Pattern).
 				Name(route.Name).
 				Handler(handler)
+
+			router.
+				Methods("OPTIONS").
+				Path(route.Pattern).
+				Name(route.Method + "-AcceptOptions").
+				Handler(AcceptOptions())
 		}
 	}
-
 	return router
 }
 
+
+func AcceptOptions() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+		w.Header().Set("Access-Control-Allow-Headers", "Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization, X-USER-ID")
+		w.Header().Set("Access-Control-Allow-Methods", "POST")
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Can you add a TODO comment here, someday we will want to lock this down, Im not sure the best course to lock it down and still allow localhost development but we can research that at a later time. Just don't want to forget about this